apps/dh/api-dh/source/DataHub.WebApi/GraphQL/GraphQLQuery.cs

```diff
@@ -36,7 +36,8 @@ public GraphQLQuery()
                 .Resolve()
                 .WithScope()
                 .WithService<IMarketParticipantPermissionsClient>()
-                .ResolveAsync(async (context, client) => await client.GetPermissionAsync(context.GetArgument<int>("id")));
+                .ResolveAsync(
```

Comment:
Question: Looks like your formatter is different that the standard one.

Reply:
hmm, have to double check, but it should just be the default Rider one